### DISCUSSION TITLE
Can not find t5_base model to download

### DISCUSSION THREAD

**Discussion by bt2513** [JohnSnowLabs/spark-nlp#10366] [Q&A] [answered]
Hi, i wrote a job using t5_base on Databricks Runtime 10.4 LTS ML with the line `T5Transformer().pretrained(name= 't5_base')` to load the model, and it was running successfully in the past few days. 
However, it stopped working 3 hours ago with the error:
`t5_base download started this may take some time.
Can not find the model to download please check the name!`

Could someone help me with this please? And is this issue going to be recurrent in the future - if so, which model should I switch to? 

It's for a question-answering task. 

My code is:
`    t5 = T5Transformer().pretrained(name= 't5_base').setTask('question:').setInputCols(["document"])`

I have `spark-nlp` installed on the cluster via PyPi, and also `com.johnsnowlabs.nlp:spark-nlp_2.12:4.0.1` via Maven. 

Thanks for your help!

**Comment by danilojsl** [comment, marked as answer]
Hi, @bt2513 I suspect this error was raised because of corruption in our metadata.json https://spark-nlp.slack.com/archives/C9YPE98AX/p1657720762994259 it was just around a week ago. Please check again and let us know
